Rakefile

```diff
@@ -56,6 +57,17 @@ task :run, :command do |t, args|
   run_command args[:command]
 end
 
+desc "Updates the rspec.github.io docs"
+task :update_docs, [:version, :branch, :website_path] do |t, args|
+  args.with_defaults(:website_path => "../rspec.github.io")
+  run_command "git checkout #{args[:branch]}", :except => UnDocumentedProjects
+  each_project :except => UnDocumentedProjects do |project|
+    cmd = "RUBYOPT='-I#{args[:website_path]}/lib' bundle exec yard --plugin rspec-docs-template --output-dir #{args[:website_path]}/source/documentation/#{args[:version]}/#{project}/"
```

Comment:
This RUBYOPT thing feels like kinda a hack but it was the best way I could figure out to make this work.  Any better ideas?